include/deal.II/cgal/intersections.h

```diff
@@ -28,6 +28,23 @@
 DEAL_II_NAMESPACE_OPEN
 namespace CGALWrappers
 {
+  template <int dim0,
+            int dim1,
+            int n_components0,
+            int n_components1,
+            int spacedim>
```

Comment:
These 5 templates are necessary to include simplices. I think we can keep the order of before, i.e. `<dim0,dim1,spacedim...`, and let `n_components0` and `n_components1` be by default the value they have for quad/hexas:`2^(dim)` .